src/server/pfs/server/master.go

```diff
@@ -202,6 +202,7 @@ func (d *driver) finishRepoCommits(ctx context.Context, repoKey string) error {
 				}); err != nil {
 					return err
 				}
+				// TODO(acohen4): this might replace an existing details object
```

Comment:
Undone TODO

Reply:
Could you elaborate on the scenario you have in mind here? Is the concern with a retry giving inaccurate details or there being a window where two PFS masters are finishing the same commit because of the properties of distributed locks. Or something else? Either way, there are a couple of rare scenarios where we currently don't get the information we probably want from this.